### PR TITLE
[FOR REVIEW] Executive Spell Copy 2023-06-28

### PR DESCRIPTION
**Pull request by LongForWisdom**

### CODE REVIEW

File: governance/votes/templates/Helpful Links.md [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 ### DC-IAM
 
-[Autoline (DC-IAM)](https://manual.makerdao.com/module-index/module-dciam)
+[Debt Ceiling Instance Access Module (DC-IAM)](https://manual.makerdao.com/module-index/module-dciam)
```
Comment (patrick-j-govalpha) [inline governance/votes/templates/Helpful Links.md:32]:
```suggestion
[Debt Ceiling Instant Access Module (DC-IAM)](https://manual.makerdao.com/module-index/module-dciam)
```

File: governance/votes/Executive vote - June 28, 2023.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+---
+
+## Proposal Details
+
+### Activate BlockTower Andromeda (RWA-015)
+
+Given the successful onboarding [poll](https://vote.makerdao.com/polling/QmbudkVR) and testing at a Debt Ceiling of 2.5 million DAI the following actions will take place if this executive proposal passes. In aggregate, these actions activate the BlockTower Andromeda vault.
+
+The [Debt Ceiling Instance Access Module (DC-IAM)](https://manual.makerdao.com/module-index/module-dciam) will be enabled for BlockTower Andromeda with the following parameters:
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - June 28, 2023.md:47]: typo

```suggestion
The [Debt Ceiling Instant Access Module (DC-IAM)](https://manual.makerdao.com/module-index/module-dciam) will be enabled for BlockTower Andromeda with the following parameters:
```

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]
See comments. Generally good

For payments etc I think it's helpful to have the addresses included since it makes it easier for anyone that is auditing the code (as they should be)

File: governance/votes/Executive vote - June 28, 2023.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+The BlockTower Andromeda oracle price will be updated to match the new Maximum Debt Ceiling.
+
+A new OutputConduit will be authorized, routing BlockTower Andromeda transactions via the USDP PSM as [confirmed](https://forum.makerdao.com/t/consolidated-action-items-for-2023-06-28-executive/21187/2) by the relevant scope facilitators. Operationally, this will mean the USDP PSM is drained to provide funding for BlockTower Andromeda.
+
+### GUSD PSM Parameter Changes
+
+As per the outcome of this recent [poll](https://vote.makerdao.com/polling/QmaXg3JT#poll-detail), the following changes to the GUSD PSM parameters will take place if this executive proposal passes.
+
+* The [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) will be reduced from 500 million DAI to **110 million DAI**. 
+* The [Fee Out](https://manual.makerdao.com/module-index/module-psm#fee-out-tout) will be reduced from 0.01% to **zero**.
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply governance/votes/Executive vote - June 28, 2023.md:61]: I don't think this is useful to add in this case (or most cases.) It's just adding complexity and more figures to misread when glancing at it quickly.

**Review by patrick-j-govalpha — APPROVED** [review]

**Pull request merged**